<?xml version="1.0"?>
<launch>
	 <!-- <node name="bhand_node" pkg="serial_bhand" type="bhand_node.py" output="screen"> -->
	 <node name="bh262_node" pkg="serial_bhand" type="bhand_node.py" output="screen">
		<param name="port" value="/dev/ttyUSB0"/>
		<param name="boud" value="9600"/>
	 </node>
	 <node name="bhand_gui" pkg="serial_bhand" type="bhand_gui.py" output="screen">
			 <param name="target_node" value="bh262_node"/>
	 </node>
</launch>
